src/main/java/com/cglx/Calculator.java

```diff
@@ -29,4 +29,8 @@ public int square(int a)
     {
         return a*a;
     }
+
+    public boolean areEqual(float f1, float f2) {
+        return f1 == f2;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Equality tests should not be made with floating point values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01:9990/coding_rules#rule_key=squid%3AS1244)